compiler/testData/diagnostics/testsWithStdLib/annotations/jvmOverloads/jvmOverloadsOnPrivate.kt

```diff
@@ -0,0 +1,17 @@
+// !DIAGNOSTICS: -UNUSED_PARAMETER
+class C {
+    <!OVERLOADS_PRIVATE!>[kotlin.jvm.overloads] private fun foo(s: String = "OK")<!> {
+    }
+}
+
+fun foo() {
+    class D {
+        <!OVERLOADS_PRIVATE!>[kotlin.jvm.overloads] fun foo(s: String = "OK")<!> {
+        }
+    }
+
+    val <!UNUSED_VARIABLE!>x<!> = object {
+        <!OVERLOADS_PRIVATE!>[kotlin.jvm.overloads] fun foo(s: String = "OK")<!> {
+    }
+    }
+}
```

Comment:
Please also add an `internal` member with `overloads` here, to ensure the warning is not reported on it